com.biglybt.BiglyBT.yaml

```
@@ -0,0 +1,56 @@
+app-id: com.biglybt.BiglyBT
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: biglybt-runner
+separate-locales: false
+finish-args:
+  - --env=PATH=/app/jre/bin:/usr/bin:/app/bin
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=fallback-x11
+  - --socket=wayland
```

does it supprot wayland? If yes
```suggestion
  - --socket=fallback-x11
  - --socket=wayland
```

---

Removed wayland and fallback-x11